pkg/dom0-ztools/rootfs/etc/containerd/config.toml

```diff
@@ -1,5 +1,5 @@
 state = "/run/containerd"
-root = "/persist/containerd"
+root = "/run/containerd-system-root"
```

Comment:
How much disk space will we use in the system root? Need to keep in mind that this consumes memory.

Reply:
Good point, thank you. I set it to another directory in persist to not consume memory.